### ISSUE TITLE
Dashboard and redesign

### ISSUE THREAD

**Issue by MallardLuna** [ionic-bot/web#118]
The webpage is fairly ugly right now and extremely limited.
I think it would be cool if we implemented support for a dashboard and redesigned it completely.

**Comment by TinyBlueSapling** [comment]
The rewrite has reached feature parity with the original website!
Should we merge to main already, or wait until the dashboard and minor details are worked out?
CC: @gamerboy80

**Comment by TinyBlueSapling** [comment]
yeah dependabot fucked up everything because i forgot to disable auto-merge on the old branch
i just renamed rewrite to main and main to old. it'll probably update a bunch of stuff now, hopefully it doesnt break everything

**Comment by MallardLuna** [comment]
I created two sub-issues to simplify this. The redesign was finished almost a year ago, but the dashboard wasn't even started.
